x-pack/plugins/security_solution/public/timelines/components/side_panel/event_details/index.tsx

```diff
@@ -159,7 +159,7 @@ const EventDetailsPanelComponent: React.FC<EventDetailsPanelProps> = ({
     setIsIsolateActionSuccessBannerVisible(true);
     // If a case details refresh ref is defined, then refresh actions and comments
     if (caseDetailsRefresh) {
-      caseDetailsRefresh.refreshUserActionsAndComments();
+      caseDetailsRefresh.refreshCase();
```

Comment:
`caseDetailsRefresh` reference has been updated to expose only one function, the implementation of the former `refreshUserActionsAndComments` has been moved to `refreshCase`.